util/syspolicy/syspolicy_test.go

```diff
@@ -67,13 +74,246 @@ func TestGetString(t *testing.T) {
 			if value != tt.value {
 				t.Fatalf("got value %v instead of expected value %v", value, tt.value)
 			}
-			if err != tt.err {
+			if err != tt.err && tt.err != ErrNoSuchKey {
```

Comment:
I feel like any time that `err != tt.err` fails is an indication that `tt.err` should be changed to the correct value, rather than letting `ErrNoSuchKey` be an exception.

I'd normally also recommend `!errors.Is(err, tt.err)` but these tests have complete control over the exact error being passed around, so the more-strict comparison works fine in this situation.